config/jobs/kubernetes/sig-node/node-docker.yaml

```diff
@@ -28,6 +28,43 @@ periodics:
   annotations:
     testgrid-dashboards: sig-node-docker
     testgrid-tab-name: docker-node-conformance
+- name: ci-docker-node-conformance-kubetest2
+  interval: 2h
+  decorate: true
+  path_alias: k8s.io/kubernetes
+  extra_refs:
+  - org: kubernetes
+    repo: test-infra
+    base_ref: master
+    path_alias: k8s.io/test-infra
+  decoration_config:
+    timeout: 65m
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+  annotations:
+    testgrid-dashboards: sig-node-docker
+    testgrid-tab-name: docker-node-conformance-kubetest2
+  spec:
+    containers:
+    - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20211108-d0d3c6d7a8-experimental
+      env:
+      - name: GOPATH
+        value: /go
+      command:
+      - runner.sh
+      args:
+      - kubetest2
+      - noop
+      - --test=node
+      - --
+      - --repo-root=.
+      - --gcp-zone=us-west1-b
+      - --parallelism=8
+      - --focus-regex=\[NodeConformance\]
+      - --skip-regex=\[Flaky\]|\[Slow\]|\[Serial\]
```

Comment:
comparing with `ci-docker-node-conformance`, why slow is also skipped?

Reply:
Nice catch, it's unintenional